src/__tests__/Home.test.js

```diff
@@ -0,0 +1,58 @@
+import { render, screen } from '@testing-library/react';
+import { Provider } from 'react-redux';
+import { BrowserRouter } from 'react-router-dom';
+import thunk from 'redux-thunk';
+import configureStore from 'redux-mock-store';
+import Home from '../components/Home';
+
```

Comment:
- [ ] You've done well on writing some tests. We however still need to write tests for the pure functions that we find in the redux code. This is as expected by this requirement 👇🏼:

  > - Testing requirements
  >   - Create unit tests for pure functions in your Redux code.
  
  Redux code is that code we have in the `redux`. In there we find some functions that help us fetch data and dispatch actions. Please let us write tests for them.

Reply:
Redux tests added